pkg/sql/backfill.go

```diff
@@ -209,6 +221,39 @@ func (sc *SchemaChanger) runBackfill(
 		}
 	}
 
+	// Make check constraints public (or drop them, if it's a rollback).
+	if len(addedChecks) > 0 || len(droppedChecks) > 0 {
+		_, err := sc.leaseMgr.Publish(ctx, sc.tableID,
+			func(desc *sqlbase.MutableTableDescriptor) error {
+				checks := make([]*sqlbase.TableDescriptor_CheckConstraint, 0)
+				for _, c := range desc.Checks {
```

Comment:
as discussed in person: I think we need to make the addition to the table desc idempotent -- either check the name, or update the mutation or something.

Reply:
Done.